cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -220,7 +219,7 @@ struct EncPage {
   uint8_t *page_data;        //!< Ptr to uncompressed page
   uint8_t *compressed_data;  //!< Ptr to compressed page
   uint16_t num_fragments;    //!< Number of fragments in page
-  uint8_t page_type;         //!< Page type (0=data, 2=dictionary)
+  PageType page_type;        //!< Page type (0=data, 2=dictionary)
```

Comment:
No need to list the values now :)
```suggestion
  PageType page_type;        //!< Page type 
```